database/scripts/generate_report.rb

```diff
@@ -27,9 +27,9 @@
 def generate_report(report_name, exclude_set)
     report = {
         'urgent' => {
-            'build_regressions' => urgent_build_regressions = BuildfarmToolsLib::build_regressions_today(filter_known: true),
-            'test_regressions_consecutive' => urgent_consistent_test_regressions = BuildfarmToolsLib::test_regressions_today(filter_known: true, only_consistent: true, group_issues: true),
```

Comment:
I think this filtering should come from a command line argument, I'm approving but we can take an extra look at this when we work on the remaining features of the report.

Reply:
Sounds reasonable